src/services/category.js

```diff
@@ -0,0 +1,9 @@
+const Category = require('~/models/category')
+
+const categoryService = {
+  getCategoriesNames: async () => {
+    return await Category.find({})
```

Comment:
1. You should delete `await`, If you're using `async,` you're already working with promises, so there's no need to add `await` before return
2. If you only need the category names, it may be worth adding only the required field to the Mongoose query. For example `Category.find({},'name')`